sylius/mollie-plugin/2.0/manifest.json

```
@@ -0,0 +1,22 @@
+{
+    "bundles": {
+        "Sylius\\MolliePlugin\\SyliusMolliePlugin": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "add-lines": [
+        {
+            "file": "assets/admin/entrypoint.js",
+            "content": "import '../../vendor/sylius/mollie-plugin/assets/admin/entry';",
```

shouldn't we have:
```suggestion
            "content": "import '../../vendor/sylius/mollie-plugin/assets/admin/entrypoint';",
```
to be consistent with the current approach (since Sylius 2.0)?